cmd/kubelet/app/options/options.go

```diff
@@ -498,6 +498,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Int32Var(&c.KubeAPIQPS, "kube-api-qps", c.KubeAPIQPS, "QPS to use while talking with kubernetes apiserver. The number must be >= 0. If 0 will use DefaultQPS: 5. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags")
 	fs.Int32Var(&c.KubeAPIBurst, "kube-api-burst", c.KubeAPIBurst, "Burst to use while talking with kubernetes apiserver. The number must be >= 0. If 0 will use DefaultBurst: 10. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags")
 	fs.BoolVar(&c.SerializeImagePulls, "serialize-image-pulls", c.SerializeImagePulls, "Pull images one at a time. We recommend *not* changing the default value on nodes that run docker daemon with version < 1.9 or an Aufs storage backend. Issue #10959 has more details.")
+	fs.Int32Var(&c.MaxParallelImagePulls, "max-parallel-image-pulls", c.MaxParallelImagePulls, "The maximum number of image pulls in parallel. The number must be >= 0. If the number is 0, then no limit will be applied. If serialize-image-pulls is true, this value will be ignored.")
```

Comment:
do we have to add command line arguments? Maybe only have it in config?

Reply:
command line arguments will be slowly deprecated one by one. I'd suggest to not include it as a command line argument so there will be no need to deprecate it later.

Reply:
gotcha, makes sense. Removed it from command line